kolibri/plugins/device/assets/src/views/ManageTasksPage/TaskPanel.vue

```diff
@@ -261,6 +265,11 @@
           '{numResources} {numResources, plural, one {resource} other {resources}} ({bytesText})',
         context: 'Indicates the number of resources and their size.',
       },
+      cancelSize: {
```

Comment:
We can't add new strings in a patch version release - and I almost think it might be better to see if we can find a way to emulate the resource import flow and be able to count the number of resources that we have successfully exported in order to be able to use the `exportSuccess` message below instead.